zproject/db/backends/postgresql/base.py

```diff
@@ -0,0 +1,11 @@
+from django.db.backends.postgresql import base
+
+if False:
+    from typing import Any
+
+class DatabaseWrapper(base.DatabaseWrapper):
+    def __init__(self, *args, **kwargs):
+        # type: (*Any, **Any) -> None
+        super(DatabaseWrapper, self).__init__(*args, **kwargs)
+        # Always log queries and track times.
+        self.force_debug_cursor = True
```

Comment:
(It'd also be good to make sure queries made via sqlalchemy are definitely included in those logs; this is what prints the server logs you get in the `run-dev.py` and production consoles).